packages/googlechrome.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,54 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+    . $toolsPath\sfta.ps1
+    . $toolsPath\helpers.ps1
+
+    $downloadDir = Get-PackageCacheLocation
+    $installer          = 'googlechromeinstaller.msi'
+    $packageArgs        = @{
+        packageName     = $env:ChocolateyPackageName
+        file            = Join-Path $downloadDir $installer
+        url             = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise.msi'
+        url64bit        = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise64.msi'
+        fileType        = 'MSI'
+        silentArgs      = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+        validExitCodes  = @(0)
+    }
+
+    # Download the installer
+    $packageArgs['file'] = Get-ChocolateyWebFile @packageArgs
+
+    $sigValid = (Get-AuthenticodeSignature -FilePath $packageArgs['file']).Status -eq 'Valid'
+
+    if ($sigValid) {
+        $toolName = 'Google Chrome'
+        $category = 'Utilities'
+
+        VM-Write-Log "INFO" "Signature valid ... installing as $category" # for lint check
+        Install-ChocolateyInstallPackage @packageArgs
+
+        VM-Write-Log "INFO" "Setup associations"
+        Set-PTA ChromeHTML http
+        Set-PTA ChromeHTML https
+        Set-FTA ChromeHTML .htm
+        Set-FTA ChromeHTML .html
```

Comment:
I think this is not working perfectly. After installing the package, opening an `.html` file shows:
![Screenshot 2023-12-12 at 12 14 22](https://github.com/mandiant/VM-Packages/assets/16052290/0a93c459-95a0-4eab-84c5-63a431d9fa39)

Reply:
my testing env seems working, after installing `cyberchef.vm`, the default opening `.html` file is Chrome without the prompt as yours.

Reply:
I have tested again in a tested environment and opening the cyberchef lnk to the html file shows this pop up. @binjo how is your testing environment set up?